tools/BuildAssets/targets/common.targets

```diff
@@ -59,6 +60,27 @@
       <ProjectRootDir/>
   </PropertyGroup>
 
+
+  <Target Name = "ImportAutoRestCodeGenerationModule" AfterTargets="Init">
+    
+    <ItemGroup>
+      <SdkBuildToolsPowershellModules Include="$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)..\psModules\CodeGenerationModules'))\**\*.*"/>
+    </ItemGroup>
+    
+    <PropertyGroup>
```

Comment:
@dsgouda format the propertygroup.
Also is there no other way but constructing this path for the user PS module directory?
This might be more work, but it's worth exploring if we can somehow get hold of this path by parsing $env:psmodulePath and using $env:userprofile to detect the path that we are interested in.

Reply:
This involves splitting `$env:psmodulePath` and check if any of the paths have a substring equal to `$env:userprofile` while this isn't difficult in itself, in msbuild it won't be straightforward. What is the harm in specifying the path as below